osf_tests/test_search_views.py

```diff
@@ -95,6 +95,80 @@ def test_search_projects(self):
         res = self.app.get(url, {'q': self.project.title})
         assert_equal(res.status_code, 200)
 
+    def test_search_user(self):
+
+        url = '/api/v1/search/user/'
```

Comment:
@brianjgeiger Was the intent for these changes to be for contributor search and not global? If so @fabmiz may need to change this around more. I don't know enough about the v1 search routes to comment.